src/services/infra/InfraService.ts

```diff
@@ -226,23 +228,28 @@ export default class InfraService {
       )
 
       // Get DNS records from Amplify
-      /**
-       * note: we wait for ard 90 sec as there is a time taken
-       * for amplify to generate the certification manager in the first place
-       * This is a dirty workaround for now, and will cause issues when we integrate
-       * this directly within the Isomer CMS.
-       * todo: push this check into a queue-like system when integrating this with cms
-       */
-      await new Promise((resolve) => setTimeout(resolve, 90000))
-
-      /**
-       * todo: add some level of retry logic if get domain association command
-       * does not contain the DNS redirections info.
-       */
+      const isTestEnv =
+        config.get("env") === "test" || config.get("env") === "dev"
+      // since we mock values during development, we don't have to await for the dns records
+      if (!isTestEnv) {
+        /**
+         * note: we wait for ard 90 sec as there is a time taken
+         * for amplify to generate the certification manager in the first place
+         * This is a dirty workaround for now, and will cause issues when we integrate
+         * this directly within the Isomer CMS.
+         * todo: push this check into a queue-like system when integrating this with cms
+         */
+        await new Promise((resolve) => setTimeout(resolve, 90000))
+        /**
+         * todo: add some level of retry logic if get domain association command
+         * does not contain the DNS redirections info.
+         */
+      }
 
       const dnsInfo = await this.launchesService.getDomainAssociationRecord(
         primaryDomain,
-        appId
+        appId,
+        subDomainSettings
```

Comment:
seems to be wrong? this method only takes 2 arguments

Reply:
9e661017 sorry, must have missed this :(